### PR TITLE
Modified wallet setup pages to include new bank info

### PR DESCRIPTION
**Pull request by starphys**

### CODE REVIEW

**Comment by MiladNazeri** [comment]
The 3 .vs files shouldn't be committed to the repo

File: 07.high-fidelity-commerce/01.basics/01.hfc/01.get-hfc/docs.md [hunk 1]
```diff
@@ -8,12 +8,8 @@ taxonomy:
...
 
-High Fidelity Commerce is in closed Beta. This means when you first set up your [Wallet](../../../marketplace/wallet-setup), you will not have any HFC available to start shopping. 
-
-You can request to be a part of the closed Beta and receive an initial grant of free HFC by emailing us at info@highfidelity.com and sending us:
-
-* Your username
-* The email address associated with your High Fidelity account. 
+High Fidelity Commerce is in open Beta. This means when you first set up your [Wallet](../../../marketplace/wallet-setup), you will not have any HFC available to start shopping. 
 
+ Want more HFC? Get it by going to [BankOfHighFidelity](https://hifi.place/BankOfHighFidelity) and meeting with the banker!
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/01.basics/01.hfc/01.get-hfc/docs.md]: turn "going to [BankOfHighFidelity]" into  "the [Bank Of High Fidelity]"

File: 07.high-fidelity-commerce/01.basics/01.hfc/02.wallet-setup/docs.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ Before you start [buying](../buy) and [selling](../sell) items in the Marketplac
 > > > > Do not lose your passphrase. Please save it or write it down, we cannot recover this for you. If you lose your passphrase, you will also lose the HFC and purchases associated with your account. 
 
 - You will be prompted to backup your private keys. This is an important step. If you lose your passphrase, we cannot recover your passphrase or your keys for you.  ![](backup-private.PNG)![](backup-inst.PNG)![](all-set.PNG)
-- You will be directed to your *Wallet Home*. Here, you can view all your recent activity and the amount of HFC you have. As commerce is in closed beta, you will need to email us at info@highfidelity.com with your High Fidelity account username and registered email address to get your starting 10,000 HFC. ![](closed-beta.PNG)  
+- You will be directed to your *Wallet Home*. Here, you can view all your recent activity and the amount of HFC you have. Want more HFC? Get it by going to [BankOfHighFidelity](https://hifi.place/BankOfHighFidelity) and meeting with the banker! ![](closed-beta.PNG)
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/01.basics/01.hfc/02.wallet-setup/docs.md]: Same thing with Bank of High Fidelity as the above comment.

File: 07.high-fidelity-commerce/chapter.md [hunk 3]
```diff
@@ -9,6 +9,6 @@ taxonomy:
 
 High Fidelity Commerce will provide a low fee cryptocurrency for use in the High Fidelity Marketplace and between users. It provides a cryptographically secure way to certify the integrity and provenance of virtual goods from that marketplace. 
 
-High Fidelity Commerce is currently in closed Beta.
+High Fidelity commerce is in open beta right now. Want more HFC? Get it by going to <a href="https://hifi.place/BankOfHighFidelity">BankOfHighFidelity</a> and meeting with the banker!
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/chapter.md]: Why is this using an anchor link? 

Please adjust to the above changes.

File: 07.high-fidelity-commerce/04.faq-section/01.commerce-faq/docs.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+<a id="q0"></a>
+**Where’s my HFC?**<br> 
+High Fidelity commerce is in open beta right now. Want more HFC? Get it by going to [Bank Of High Fidelity](https://hifi.place/BankOfHighFidelity) and meeting with the banker!
+<p align="right"><a href="#top">Back to Top</a></p>
+<br>
+
+<a id="q1"></a>
+**What are private keys and where are they stored?**<br> 
+A private key is a secret piece of text that is used to prove ownership, unlock confidential information, and sign transactions. In High Fidelity, your private key is used to securely access the contents of your Wallet and Purchases. After wallet setup, a hifikey file is stored on your computer in High Fidelity Interface's AppData directory. Your hifikey file contains your private key and is protected by your wallet passphrase.
+It is very important to backup your hifikey file! To find the folder where your hifikey file is stored, navigate to the Help section in Interface. ![](HiFiPrivateKeyStorage.PNG)
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/04.faq-section/01.commerce-faq/docs.md]: "hifikey" <-- that looks funny, is this a typo?  hifi-key?

File: 07.high-fidelity-commerce/04.faq-section/01.commerce-faq/docs.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+**What is a Security Pic?**<br>
+Your Security Pic is an encrypted image that you select during Wallet Setup. It acts as an extra layer of Wallet security. When you see your Security Pic, you know that your actions and data are securely making use of your private keys. 
+
+Don’t enter your passphrase anywhere that doesn’t display your Security Pic! If you don't see your Security Pic on a page that requests your Wallet passphrase, someone untrustworthy may be trying to access your Wallet. 
+<p align="right"><a href="#top">Back to Top</a></p>
+<br>
+
+<a id="q7"></a>
+**My HFC balance doesn’t update instantly. Why?**<br> 
+HFC transactions sometimes takes a few seconds to update as they are backed by a blockchain. [Learn more here](https://docs.highfidelity.com/high-fidelity-commerce).
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/04.faq-section/01.commerce-faq/docs.md]: I think this is take not takes

File: 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+| [Why use HFC and not just use another cryptocurrency like Ethereum?](#q14) |
+| [What is Federated Consensus?](#q15)     |
+| [Can I get paid miner’s fees?](#q16)     |
+| [What are transaction fees?](#q17)       |
+| [What is High Fidelity's approach to Know Your Customer, Anti-Money Laundering, and sales tax? ](#q18) |
+<a id="q0"></a>
+<br>
+
+
+###General Questions
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md]: space after ###

File: 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+<a id="q14"></a> 
+**If I lose my wallet and my wallet passphrase, Why can't High Fidelity recover it?**<br> 
+High Fidelity has no record of what "you" (or your username, avatar, phone number, etc.) have bought or been paid. Although the blockchain ledger is available for everyone to inspect, the participants are all recorded as anonymous keys. Your wallet contains these keys, and it is entirely up to you to make your own backup of the wallet file. 
+High Fidelity cannot spend, observe, share, lose, or recover your wallet, passphrase, keys, HFC, or purchased items.
+<p align="right"><a href="#top">Back to Top</a></p>
+<br>
+
+
+
+###Blockchain Architecture Questions
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md]: space after ###

File: 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+<br>
+<a id="q15"></a>
+**Why use HFC and not just use another cryptocurrency like Ethereum or ERC-20 Tokens?  **<br>
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md]: You can't have a space before that last **

File: 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+<a id="q15"></a>
+**Why use HFC and not just use another cryptocurrency like Ethereum or ERC-20 Tokens?  **<br> 
+HFC transactions will have very low costs, where as other currency transactions typically cost more money than we anticipate users wishing to spend on an individual digital good. Controls over transaction costs are not supported by Ethereum and other networks.  
+Separately, the speed of transactions on the HFC network is much higher than what is available on a fully distributed network, as we anticipate needing to support a real-time marketplace for the purchase of digital goods.
+Finally, the price volatility of BTC and ETC combined with their generally upward price trend makes them a poor choice as a trading currency — they have become more like decentralized commodities. HFC can be easily exchanged for these currencies when needed, but in-world transactions happen against the faster consensus network.
+<p align="right"><a href="#top">Back to Top</a></p>
+<br>
+
+<a id="q16"></a>
+**What is Federated Consensus? **<br>
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md]: Same here with the space and the **

File: 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+<p align="right"><a href="#top">Back to Top</a></p>
+<br>
+
+<a id="q17"></a> 
+**Can I get paid miner’s fees? **<br>
```
Comment (MiladNazeri) [inline 07.high-fidelity-commerce/04.faq-section/02.digital-asset-registry-faq/docs.md]: Same here

File: 07.high-fidelity-commerce/04.faq-section/01.commerce-faq/docs.md (hunk 4 above)
Comment (starphys) [reply 07.high-fidelity-commerce/04.faq-section/01.commerce-faq/docs.md]: .hifikey is the file type

**Pull request merged**